src/vi/hentaivn/src/eu/kanade/tachiyomi/extension/vi/hentaivn/HentaiVN.kt

```diff
@@ -103,7 +103,7 @@ class HentaiVN : ParsedHttpSource() {
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select(".main > .page-left > .left-info > .page-info")
         val manga = SManga.create()
-        manga.title = infoElement.select("h1[itemprop=name] a").text()
+        manga.title = infoElement.select("h1[itemprop=name] a").text().replaceAfter("-", "").replace("-", "").trim()
```

Comment:
```suggestion
        manga.title = document.selectFirst("li[itemprop=itemListElement]:last-child span[itemprop=name]").text()
```

the original title is in the breadcrumb above, no need for string processing.

![image](https://user-images.githubusercontent.com/92439990/213908560-abbe5566-445e-4be9-9224-dd68137f3c23.png)